en/Obsidian Sync/Limitations.md

```diff
@@ -18,9 +18,9 @@ At the moment, each file can be up to 100 MB.
 
 ## How long do you keep my data after my subscription expires?
 
-We keep data in your remote vaults, including version history, for one month after your subscription expires.
+We keep data in your remote vaults, including version history, for one month after your subscription expires. Your local vaults, stored on your devices, will not be affected.
 
-As long as you renew within one month, there should be no impact on your usage.
+As long as you renew within one month, there should be no impact on your usage. In case you don't and your remote vaults no longer exist, you can easily [[Set up Obsidian Sync|create a new remote vault and re-connect your local vault]] again.
```

Comment:
```suggestion
As long as you renew within one month, there should be no impact on your usage. If you renew after a month, when your remote vaults have been removed, you can [[Set up Obsidian Sync|create a new remote vault re-connect your local vault]].
```

Reply:
- Change from negative to positive statement.
- "again" is redundant with "RE-connect".